.travis.yml

```diff
@@ -2,21 +2,19 @@ language: php
 
 matrix:
   include:
-  - php: 7
+  - php: 7.1
```

Comment:
You want to switch to 7.1 only or would it be worthwhile to prove 7.0 is working as well?